apps/mochi-web/pages/_app.tsx

```diff
@@ -21,7 +21,6 @@ const WalletProvider = dynamic(() =>
 const LoginWidgetProvider = dynamic(() =>
   import('@mochi-ui/core').then((m) => m.LoginWidgetProvider),
 )
-const Toaster = dynamic(() => import('sonner').then((m) => m.Toaster))
```

Comment:
Should we remove the package from package.json?

Reply:
Also, can you make sure there is no significant change in the UI here? If the difference is notable, please keep the old one.

Reply:
We still use this package so I have not removed it yet